index.html

```diff
@@ -805,6 +818,16 @@
 and tend to be [=legitimate=]. When it is done in the [=person=]'s stead and without [=governance=],
 it decreases personal [=autonomy=] and tends to be [=illegitimate=].
 
+<!--
+XXX this needs a better formulation
+* the data pertains to a group of people such that a [=person=] may find
+  themselves to be the subject of a treatment related to this group, even if the
+  entity carrying out the treatment has no way to identify that [=person=].
+
+We also need to surface surveillance vs capture -> industrialisation of society -> origin sovereignty
+as the primary line of defence against collective privacy harms from capture/enclosure/control
+-->
+
```

Comment:
This probably isn't part of this PR, but it'd be worth capturing it as an issue.